gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -64,6 +77,20 @@ export default function NxFormLayoutExample() {
         setFilePristine(false);
       };
 
+  const { initialState, userInput } = nxTextInputStateHelpers,
+      [matches, setMatches] = useState<DataItem<number, string>[]>(comboboxItems),
+      [inputState, setInputState] = useState(initialState('')),
```

Comment:
There are many inputs in this example, let's use a more distinctive name for this variable

Reply:
Updated here 8d3cb2ac97adaad4eccc60e1608d4414f357cb8b.